### PR TITLE
(MAINT) Update CHANGELOG for 0.4.1 release

### PR DESCRIPTION
**Pull request by nwolfe**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+## 0.4.1
+ This is a maintenence release.
```
Comment (camlow325) [inline CHANGELOG.md]: s/maintenence/maintenance

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,9 @@
...
+
+ * Add documentation for making requests using the Java and Clojure clients.
+ * Upgrade jvm-certificate-authority dependency to 0.7.0.
```
Comment (camlow325) [inline CHANGELOG.md]: Maybe mention the bump for the ks and trapperkeeper deps too

**Comment by camlow325** [comment]
:+1: with minor comments...

**Comment by nwolfe** [comment]
@camlow325 updated inline

**Comment by camlow325** [comment]
:+1:

**Pull request merged**